docs/image/upload-image.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: upload-image
-sidebar_position: 6
```

Comment:
Since you changed upload-image.md path. The `Storage Class` link becomes `
```
https://66e7fba694e4b1faa71c4190--harvester-preview.netlify.app/v1.4/image/advanced/storageclass.md
``` 
which is not found.